### DISCUSSION TITLE
Add “.” (dot) option

### DISCUSSION THREAD

**Discussion by JohnnyX89** [BitCannaGlobal/WeedMobile#90] [Ideas]
When establishing the amount to be send there's no option to be put a "." for decimals so only full values (coins) can be send or requested when creating the QR code.

![image](https://github.com/BitCannaGlobal/WeedMobile/assets/53493683/1a767de1-ba14-429f-b0ec-f0f0b41abe21)

**Comment by RaulBernal** [comment]
I think this issue is related with your device.
We have tested in other iOS devices and works fine, for example:

- iOS iPad
![telegram-cloud-photo-size-4-5985501703023214209-y](https://github.com/BitCannaGlobal/WeedMobile/assets/3751926/1eb0ba28-0a2a-45bd-aa68-703fdc4dba6d)

- iOS iMac (as desktop App)
![telegram-cloud-photo-size-4-5985501703023214210-y](https://github.com/BitCannaGlobal/WeedMobile/assets/3751926/a8e1bca8-2dee-44d5-9bb2-854c94e39062)

- Android device:
![telegram-cloud-photo-size-4-5985501703023214205-y](https://github.com/BitCannaGlobal/WeedMobile/assets/3751926/2e9a02e7-7250-4cb2-84a7-5ace7b84d848)

**Reply by JohnnyX89** [reply]
Is possible to add it, but inside the app there's no "." option when selecting the amount field on the pop up keyboard (initial print)

I can add it manually by copy/paste a . from any other field

**Comment by RaulBernal** [comment]
Let me know your iOS version and device model please, you can send me by
private if you don't want to disclose it here 😄

**Reply by JohnnyX89** [reply]
iPhone 14 - iOS version 17.3.1 (21D61)
